kubetest/e2e.go

```diff
@@ -623,10 +623,35 @@ func kubemarkTest(testArgs []string, dump, numNodes string) error {
 		})
 	}
 
+	if err := control.XmlWrap(&suite, "kubectl version", getKubectlVersion); err != nil {
+		return err
+	}
+
+	if err := control.XmlWrap(&suite, "IsUp", deploy.IsUp); err != nil {
+		return err
+	}
+
 	// Run tests on the kubemark cluster.
 	if err := control.XmlWrap(&suite, "Kubemark Test", func() error {
-		testArgs = util.SetFieldDefault(testArgs, "--ginkgo.focus", "starting\\s30\\pods")
-		return control.FinishRunning(exec.Command("./test/kubemark/run-e2e-tests.sh", testArgs...))
+		testArgs = append(
+			util.SetFieldDefault(testArgs, "--ginkgo.focus", "starting\\s30\\pods"),
```

Comment:
Does this mimic this logic correctly - https://github.com/kubernetes/kubernetes/blob/master/test/kubemark/run-e2e-tests.sh#L35-L39?

Reply:
and seems --test-args is always non-empty for CI jobs, I'm dropping the default logic. I can add that back.